PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx

```diff
@@ -111,6 +114,10 @@ end
 
 function MasterControl:SetIsJumping(jumping)
 	isJumping = jumping
+	local humanoid = self:GetHumanoid()
+	if humanoid and not humanoid.PlatformStand then
+		humanoid.Jump = isJumping
```

Comment:
I believe so - in SetIsJumping, humanoid.Jump is being set independent of the humanoid's state, so that JumpRequest reliably fires whenever the jump button is pressed. In renderstep, it only gets set if the player is in state that allows them to jump, other wise MasterControl is fighting with the engine as to whether humanoid.Jump should be true, and JumpRequest generally fires 2 or 3 times on each jump. So you need to set it in both.